learn/get-started/basics.md

```diff
@@ -13,7 +13,7 @@ In this video, we walk you through setting up Visual Studio Code and give an ove
 
 <iframe src="https://www.youtube-nocookie.com/embed/ITxcbrfEcIY" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen title="Getting Started with Visual Studio Code"></iframe>
 
-Next video: [Extensions for Visual Studio Code](/learn/get-started/extensions.md)
+Next video: [Extensions for Visual Studio Code](/training/get-started/extensions.md)
```

Comment:
This is incorrect. Please revert any places where relative topic links to content under /learn were updated to /training.

Reply:
The linked file looks like it's local to the same directory as this file. Would you prefer I change these to local links instead?

Reply:
Please don't change any relative links to vscode-docs files. 
You should only be changing absolute URLs that go to the root https://docs.microsoft.com.
The vscode-docs content uses relative content links extensively, which allows people to navigate the Markdown files on GitHub as well as the rendered URLs on code.visualstudio.com. 
For example, the relative link above goes to https://github.com/microsoft/vscode-docs/blob/main/learn/get-started/extensions.md and is rendered as a link to https://code.visualstudio.com/learn/get-started/extensions.
We also prefer to use paths relative to the vscode-docs root so that links will still work even if a topic containing the relative link is moved.

Reply:
Reverted